new-lamassu-admin/src/pages/Triggers/helper.js

```diff
@@ -119,24 +108,66 @@ const Schema = Yup.object()
     threshold
     // direction
   })
-  .test(
-    'are-fields-set',
-    'Invalid values',
-    ({ threshold, triggerType }, context) => {
-      const validator = {
-        txAmount: threshold => threshold.threshold >= 0,
-        txVolume: threshold =>
-          threshold.threshold >= 0 && threshold.thresholdDays > 0,
-        txVelocity: threshold =>
-          threshold.threshold > 0 && threshold.thresholdDays > 0,
-        consecutiveDays: threshold => threshold.thresholdDays > 0
-      }
-      return (
-        (triggerType && validator?.[triggerType](threshold)) ||
-        context.createError({ path: 'threshold' })
-      )
+  .test(({ threshold, triggerType }, context) => {
+    const errorMessages = {
+      txAmount: threshold => 'Amount must be greater than or equal to 0',
+      txVolume: threshold => {
+        const thresholdMessage = 'Volume must be greater than or equal to 0'
+        const thresholdDaysMessage = 'Days must be greater than 0'
+        let message = ''
```

Comment:
Maybe it's better to have message be a array and then join the array with a , as separator. With this we can ditch the let for a const.